README.md

```
@@ -112,6 +146,15 @@ To complete the signup portal deployment below MOSIP kernel services are require
     *  Make sure to update the `signup-keystore-password` in the keycloak secrets as updated while generating the p12 file.
 5. Mount oidckeystore.p12 as a `signup-keystore` secret to the **signup deployment**.
 
+## TESTRIG
+
+## Overview
+The steps here install Testrig components, which include SIGNUP-APITESTRIG .
+These components are used to test the working of SIGNUP modules.
+
+1.SIGNUP-APITESTRIG Installation
+[signup-apitestrig ](deploy/signup-apitestrig/README.md)
```

markdownlint-cli2 (0.18.1)

Spaces inside link text

(MD039, no-space-in-links)